src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/machines/Freezer.java

```diff
@@ -17,16 +18,26 @@
 
 public class Freezer extends AContainer implements RecipeDisplayItem {
```

Comment:
Can this class have javadocs please?
Not in the scope of this PR, I know, just pointing it out